code/modules/events/infestation.dm

```diff
@@ -6,7 +6,7 @@
 #define LOC_VAULT 5
 #define LOC_CONSTR 6
 #define LOC_TECH 7
-#define LOC_TACTICAL 8
+#define LOC_GARDEN 8
```

Comment:
I wonder; why bother even using these defines? Just for dev QoL? Can't you accomplish the same feat with a simple integer and no define attached to it?